servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block.evictor;
+
+import java.io.File;
+import java.util.Collections;
+
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.worker.block.BlockMetadataManager;
+import tachyon.worker.block.BlockMetadataManagerView;
+import tachyon.worker.block.TieredBlockStoreTestUtils;
+import tachyon.worker.block.meta.StorageDir;
+
+/**
+ * Base class for unit tests of evictors.
+ *
+ * It provides some utilities and initializes an {@link Evictor}, a {@link BlockMetadataManager} and
+ * a {@link BlockMetadataManagerView} for a default tiered storage defined in
+ * {@link TieredBlockStoreTestUtils#defaultTachyonConf(String)}.
+ */
+public class BaseEvictorTest {
+  protected static final int USER_ID = 2;
+  protected static final long BLOCK_ID = 10;
+
+  protected BlockMetadataManager mMetaManager;
+  protected BlockMetadataManagerView mManagerView;
+  protected Evictor mEvictor;
+
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+
+  /**
+   * Cache a block to the tiered storage managed by the {@link #mMetaManager}. It's a wrapper around
+   * {@link TieredBlockStoreTestUtils#cache}.
+   *
+   * @param userId ID of user to cache this block
+   * @param blockId ID of the block
+   * @param bytes length of the block in bytes
+   * @param tierLevel tier level for the block in the tiered storage
+   * @param dirIndex directory index in tierLevel for the block in the tiered storage
+   * @throws Exception when anything goes wrong, should not happen in unit tests
+   */
+  protected void cache(long userId, long blockId, long bytes, int tierLevel, int dirIndex)
+      throws Exception {
+    StorageDir dir = mMetaManager.getTiers().get(tierLevel).getDir(dirIndex);
```

Comment:
Got what you mean, will add an overloaded method with parameters like this back to `TieredBlockStoreTestUtils`.